community-packages/package-list.json

```diff
@@ -227,6 +227,10 @@
     {
       "repoSlug": "pulumiverse/pulumi-matchbox",
       "schemaFile": "provider/cmd/pulumi-resource-matchbox/schema.json"
+    },
+    {
+      "repoSlug": "lbrlabs/pulumi-lbrlabs/eks",
```

Comment:
I believe this should be `lbrlabs/pulumi-lbrlabs-eks`

Reply:
@jaxxstorm You'll also need to change the `layout` param in your `_index.md` doc to `package`. (Apparently this was changed recently so just calling it out so you aren't surprised when the docs fail to render.) We should also get a bit more content into the examples in those docs before merging. (Examples look TBD-ish.)

Reply:
ah yes, good catch!